<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mapper_U2</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPlainTextEdit" name="console">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QTabWidget" name="tools">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>U2</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Draw Action</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QPushButton" name="btn_drw_none">
            <property name="text">
             <string>None</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">draw_group</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_drw_points">
            <property name="text">
             <string>Points</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">draw_group</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>U2 - Convex hull</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="3" column="1">
           <widget class="QComboBox" name="cmb_rand_shape">
            <item>
             <property name="text">
              <string>Circle</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Square</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Random</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Raster</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QSpinBox" name="spn_rand_points">
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>1000000</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QPushButton" name="btn_gen_convex_hull">
            <property name="text">
             <string>Generate  convex hull</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QComboBox" name="cmb_convex_hull_alg">
            <item>
             <property name="text">
              <string>Jarvis Scan</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Quick Hull</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sweep Line</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Graham Scan</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QPushButton" name="btn_gen_rand_points">
            <property name="text">
             <string>Generate points</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QPushButton" name="btn_tst_algorithms">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Algorithms Test</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_clear">
         <property name="text">
          <string>Clear all</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="Draw" name="canvas" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>5</horstretch>
       <verstretch>5</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Draw</class>
   <extends>QWidget</extends>
   <header>draw.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="draw_group"/>
 </buttongroups>
</ui>
